python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -9,71 +9,34 @@ the output for central dispersion analysis.
 
 Let us denote by :math:`\uX` the input random vector.
 Assume that the marginals of :math:`\uX` are independent.
-Suppose that :math:`Z = h(\ux)` is a real function of the
-input, i.e. :math:`n_Z = 1`.
-Assume that the order 1 Taylor expansion of the function :math:`h`
-at the point :math:`\ux = \muX` is exact, i.e.:
 
-.. math::
-    h(\ux) = h(\muX)
-    + \sum_{i=1}^{n_X} \frac{\partial h}{\partial x_i} (\muX) (x_i - \mu_i)
-
-where:
+We note :math:`Y = h(\ux)` with :math:`h: \Rset^d \rightarrow \Rset`.
 
--  :math:`\muX` is the mean of the input random vector,
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations:
 
--  :math:`\frac{\partial h}{\partial x_i} (\muX)`
-   is the partial derivative of the model :math:`h`
-   with respect to the i-th input variable,
-   evaluated at the point :math:`\ux`.
-
-Therefore the expectation of :math:`Z` is:
+The first order  Taylor expansion of :math:`h` around :math:`\ux = \mu = \Expect{\uY}` yields to the first order approximation of :math:`\Expect{Y}`:
```

Comment:
```suggestion
The first order  Taylor expansion of :math:`h` around :math:`\ux = \mu = \Expect{\uY}` yields the first order approximation of :math:`\Expect{Y}`:
```

Reply:
I modified the text and the sentence is no longer there.